packages/grafana-icons/package.json

```diff
@@ -16,6 +16,7 @@
   "types": "src/index.ts",
   "publishConfig": {
     "main": "dist/index.js",
+    "module": "dist/index.js",
```

Comment:
Without this addition we would need to allow overrides in the esmOutput function but that seems odd to me as esm packages should declare a module property in their package.json.